cli/cmd/plugin/ui/web/tanzu-ui/src/shared/utilities/Array.util.ts

```diff
@@ -12,6 +12,23 @@ export function middle<ARRAY_TYPE>(source: ARRAY_TYPE[]): ARRAY_TYPE | undefined
     return elementAt(source, index);
 }
 
+/**
+ * Find the first element in fallbacks which is also in options.
+ * @param options is a list of string.
+ * @param fallbacks contains all fallbacks which have 0 or 1 element in the options.
+ * @returns string is the element in both options and fallbacks.
+ */
+export function findFirstFallbackOption(options: string[], fallbacks: string[]): string {
```

Comment:
Since this is a generic array utility I think we should rename and update comments to not use 'fallback' but instead use 'match.' So function name is findFirstMatchInLists or something like that